test-deployed-function.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+set -e          # Fail on any error
+set -o pipefail # Ensure piped commands propagate exit codes properly
+set -u          # Treat unset variables as an error when substituting
+
+# Load the project variables
+source ./set-project-vars.sh
+
+echo "🌀 Emitting a Pubsub event to trigger the function..."
+printf "\n"
+gcloud pubsub topics publish "${topic_name}" --message='{"rate_feed_name": "TEST/USD", "relayer_address": "0x13374935239dacdecf7c5ba76d8de40b077b7420"}'
```

Comment:
Let's add a TODO in here so that we don't forget to update this to an actual relayer address later?

Reply:
👍